src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -757,12 +762,25 @@ static function (ChildDefinition $definition, AsPeriodicTask|AsCronTask $attribu
                 }
             );
         }
-        $container->registerAttributeForAutoconfiguration(JsonStreamable::class, static function (ChildDefinition $definition, JsonStreamable $attribute): void {
-            $definition->addTag('json_streamer.streamable', [
+
+        $container->registerForAutoconfiguration(CompilerPassInterface::class)
+            ->addExcludeTag('container.excluded.compiler_pass');
+        $container->registerForAutoconfiguration(KernelInterface::class)
+            ->addExcludeTag('container.excluded.kernel');
+        $container->registerForAutoconfiguration(TestCase::class)
+            ->addExcludeTag('container.excluded.test_case');
+        $container->registerAttributeForAutoconfiguration(AsMessage::class, static function (ChildDefinition $definition) {
+            $definition->addExcludeTag('container.excluded.messenger.message');
+        });
+        $container->registerAttributeForAutoconfiguration(Entity::class, static function (ChildDefinition $definition) {
```

Comment:
This one should be in the `DoctrineBundle`, there is nothing related to Doctrine in the FrameworkBundle.
See https://github.com/doctrine/DoctrineBundle/pull/1868 that includes other Doctrine attributes, and https://github.com/doctrine/DoctrineMongoDBBundle/pull/876.

Reply:
See https://github.com/symfony/symfony/pull/59987#discussion_r1997660249

Reply:
Thanks, my page wasn't up to date with the review.
So it's done in FrameworkBundle, there is also `Embeddable` and `MappedSuperclass` to exclude.

Reply:
See my comment here: https://github.com/symfony/symfony/pull/59987#discussion_r1997660249